invenio_records_rest/serializers/json.py

```diff
@@ -25,8 +25,8 @@ def _format_args():
         # Ensure we can run outside a application/request context.
         try:
             pretty_format = \
-                current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and \
-                not request.is_xhr
+                current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] or \
```

Comment:
I think by default we should not prettyprint the JSON (to avoid adding lots of spaces/tabs into the output). Perhaps, it's better if we just add a request parameter that you can use instead.....e.g. ``/api/records/?prettyprint``

Reply:
The default value for the config was changed to `False` in Flask v0.12, (so no prettyprint by default). So in the end, which of the following should we use?

- [ ] `JSONIFY_PRETTYPRINT_REGULAR`
- [ ] `current_app.debug`
- [ ] `'prettyprint' in request.args`

Reply:
``if request and request.args.get('prettyprint')``? 

I think it's still useful to have the JSON formatted nicely for display in a browser.....

Reply:
so simply remove debug and ``JSONIFY_PRETTYPRINT_REGULAR``